src/util/trackingUtil.js

```diff
@@ -80,7 +80,7 @@ export const getAllDimensions = (
   const dimensions = {
     4: contentTypeLabel,
     5: subject ? subject.name : undefined,
-    6: topicPath ? [0]?.name : undefined,
+    6: topicPath[0]?.name || undefined,
```

Comment:
Er det gitt at topicPath ikkje er undefined?
```suggestion
    6: topicPath?.[0]?.name || undefined,
```

Reply:
Det kan den vel være ja...